mod_conf_sql.html

```diff
@@ -195,6 +213,11 @@
   proftpd -c sql://foo:bar@localhost/db:proftpd/ctxt:ftpctxt/conf:ftpconf/map:ftpmap/base_id=<i>baseid</i>
 </pre>
 
+<p> for sqlite dbuser and dbpass are note used, use dbserver for file path and use \ in place of / for the file path : </p>
+<pre>
+  proftpd -c sql://nouser:nopass@\path\to\sqlite.db/db:sqlite/ctxt:ftpctxt/conf:ftpconf/map:ftpmap
```

Comment:
or using uri encoding...

using %2F in place of / in sqlite file path, like that y replace my replace code that replace \ in / with replacing %2F in /...

```
sql://nouser:nopass@%2Fpath%2Fto%2Fsqlite%2Fdb/db:sqlite/ctxt:ftpctxt/conf:ftpconf/map:ftpmap
```